mobile/src/main/java/com/gelakinetic/mtgfam/helpers/database/CardDbAdapter.java

```diff
@@ -796,6 +796,18 @@ public static Cursor Search(SearchCriteria criteria, boolean backface, String[]
         List<String> supertypes = criteria.superTypes;
         List<String> subtypes = criteria.subTypes;
 
+        if(criteria.isCommander){

```

Comment:
I haven't compiled and tested, but I don't think this logic is sufficient. Does it return [Ormendahl, Profane Prince](https://scryfall.com/card/soi/281/westvale-abbey-ormendahl-profane-prince?back), an illegal commander?

Reply:
That's a case I hadn't thought of.  In the same vein, Withengar Unbound also appears.

Reply:
Edge cases are tough. I think you just need to shoehorn in a check that the card's number doesn't contain a 'b'.

Reply:
Is there a simple way of checking if a card is the front side? I'm not too familiar with the back- end here.

Reply:
Every card has a number (well maybe not Vanguards, blech) for ```KEY_NUMBER```. That data is actually a string. For cards with only one face, it's just a number, like 123. For cards that have two faces, one will have 123a (front) and the other will have 123b (back). For cards like "Fire // Ice" the front / back distinction doesn't apply. So just make sure there isn't a 'b' in the number and I think that'll filter out backface legendary creatures.

Reply:
I have that fixed, but now I need to allow planeswalker commanders.  Somehow that is a simple case which slipped my mind.

Reply:
That is another good case which I didn't think about either.

Reply:
And also [The Grand Calcutron](https://scryfall.com/card/ust/131/the-grand-calcutron):

> In Commander games allowing silver-bordered cards, The Grand Calcutron can be your commander.
> (2018-01-19)

Reply:
Should I have it to where silver border is allowed? Currently it filters out any and all cards not legal in edh. Other than that, all of the issues are accounted for. (double checked with scryfall search "https://scryfall.com/search?q=is%3Acommander+-e%3Ac19+f%3Aedh&unique=cards&as=grid&order=name" (C19 not in app's database)).